control/xferfcn.py

```diff
@@ -1130,21 +1130,18 @@ def sample(self, Ts, method='zoh', alpha=None, prewarp_frequency=None,
         copy_names : bool, Optional
             If True, copy the names of the input signals, output
             signals, and states to the sampled system.
-
-        Returns
-        -------
-        sysd : TransferFunction system
-            Discrete-time system, with sample period Ts
-
-        Additional Parameters
-        ---------------------
```

Comment:
As above, this should be named "Other Parameters" and should go _after_ the "Returns" section.